web/.prettierrc

```diff
@@ -1,6 +1,8 @@
-printWidth: 75
-semi: false
-singleQuote: true
-tabWidth: 4
-trailingComma: 'none'
-bracketSpacing: true
+{
```

Comment:
This didn't work for me, without an extension it had to be yaml

Reply:
I got an error without the extension (it was expecting json) so I just added .yaml? Does that work?